### ISSUE TITLE
[Bug]: Select triggers on touch scroll

### ISSUE THREAD

**Issue by ChrisGV04** [radix-vue/radix-vue#804]
### Environment

```bash
- Operating System: Darwin
- Node Version:     v18.17.1
- Package Manager:  pnpm@8.15.4
- Radix Vue version: 1.6.1
- Browser: Chrome 123
```

### Link to minimal reproduction

https://www.radix-vue.com/components/select.html

### Steps to reproduce

On touch devices, try to scroll up/down by starting the scroll gesture on top of the Select trigger.

### Describe the bug

If you start a scroll gesture, the `Select` gets triggered and opens the `SelectContent`, which feels wrong. Any other component if you start the gesture it doesn't trigger until an actual click/tap happens.

I believe the issue comes from the `SelectTrigger` which listens to the `@pointerdown` event and on touch devices it immediately triggers the event even on scroll.

I see that even the official Radix UI has the same problem.

### Expected behavior

I would expect that the `Select` only gets triggered on click, because if I'm trying to scroll the page starting from the trigger, it blocks scroll and opens the options, which doesn't feel right.

### Context & Screenshots (if applicable)

https://github.com/radix-vue/radix-vue/assets/47041342/6a6cf580-6192-49d1-a3b5-2516b980fe63

**Comment by MellKam** [comment]
It seems like `radix-ui` also has this issue


https://github.com/radix-vue/radix-vue/assets/51422045/5e1d2ca6-e331-44d4-bab0-2c32916c214b

**Comment by ChrisGV04** [comment]
Indeed. And I'm pretty sure that it's because they both use the `@pointerdown` event to open the content instead of `@click`.

I just don't understand exactly what's the reason behind that choice. Even other components that have similar behaviors don't do it like that.

The `Popover` and `DropdownMenu` don't have the issue because they do trigger `@click`.

**Comment by ChrisGV04** [comment]
I have been playing for a while now trying to get the interaction to work as expected but it seems to be more complicated than I thought... Here are the options I tried:

### Merge `@pointerdown` to `@click`
This fixes the scroll activation, but for some reason this caused selecting an item with the `Enter` key to not close the menu.

### Replace `@pointerdown` with `@pointerup`
This also fixes the scroll activation, but now sometimes pressing on a `SelectItem` with touch devices doesn't select it until a second press. I have no idea why that happens. Reading MDN makes me think that using pointer events requires more control.

**Comment by ChrisGV04** [comment]
I think I have a solution... Will submit PR in a few minutes
